docs/environment_variables.md

```diff
@@ -2,6 +2,13 @@ Environment Variables
 =====================
 
 The HTTPX library can be configured via environment variables.
+To activate the environment variables, `trust_env` has to be set `true`.
```

Comment:
Environment variables are used by default, we should mention that it can be disabled via trust_env=False